### PR TITLE
All sites: revisit CSS prefixes

### PR DESCRIPTION
**Pull request by arthurvr**
I got over every prefixed CSS property and revisited for our current browser support. This would fix https://github.com/jquery/jquery-wp-content/issues/335.

### CODE REVIEW

File: themes/jquery/css/base.css [hunk 1]
```diff
@@ -477,8 +477,6 @@ input[type="radio"] {
 
 input[type="search"] {
 	-webkit-appearance: textfield; /* 1 */
-	-moz-box-sizing: content-box;
-	-webkit-box-sizing: content-box; /* 2 */
```
Comment (scottgonzalez) [inline themes/jquery/css/base.css]: Remove the associated comment above as well.

File: themes/jquery/css/base.css [hunk 2]
```diff
@@ -1730,9 +1700,7 @@ nav#main {
 	border-right: 1px solid rgba(2, 2, 2, 0.28);
 	border-left: 1px solid rgba(2, 2, 2, 0.28);
 	border-top: 1px solid rgba(250, 250, 250, 0.27);
-	-webkit-box-shadow: rgba(255,255,255,0.3) 0 1px 0, rgba(0,0,0,0.3) 0 -1px 0;
-	box-shadow: rgba(255,255,255,0.3) 0 1px 0, rgba(0,0,0,0.3) 0 -1px 0;
-	-webkit-box-shadow: 0 0 3px rgba(1, 1, 1, 0.7);
+  box-shadow: rgba(255,255,255,0.3) 0 1px 0, rgba(0,0,0,0.3) 0 -1px 0;
```
Comment (scottgonzalez) [inline themes/jquery/css/base.css]: bad indentation

File: themes/learn.jquery.com/style.css [hunk 3]
```diff
@@ -44,34 +44,26 @@ a {
...
 
 .beta-ribbon:hover {
 	background-color: #ffea5e;
-	background-image: -webkit-gradient(linear, left top, left bottom, from(#ffea94), to(#FFEA5E)); 
 	background-image: -webkit-linear-gradient(top, #ffea5e, #FFea94); 
-	background-image:    -moz-linear-gradient(top, #ffea5e, #FFea94); 
-	background-image:     -ms-linear-gradient(top, #ffea5e, #FFea94); 
-	background-image:      -o-linear-gradient(top, #ffea5e, #FFea94); 
+	background-image: linear-gradient(to bottom, #ffea5e, #FFea94); 
+
```
Comment (scottgonzalez) [inline themes/learn.jquery.com/style.css]: extraneous blank line

**Comment by arthurvr** [comment]
@scottgonzalez Updated PR. Thanks for the review!

**Pull request closed without merging**